x-pack/test/security_solution_cypress/config.ts

```diff
@@ -46,6 +46,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
         // See https://github.com/elastic/kibana/pull/125396 for details
         '--xpack.alerting.rules.minimumScheduleInterval.value=1s',
         '--xpack.ruleRegistry.unsafe.legacyMultiTenancy.enabled=true',
+        '--server.publicBaseUrl=http://localhost:5620',
```

Comment:
Hi @michaelolo24! It is preferred to don't commit this type of local configurations. So please, remove it when you can. Thank you 😃

Reply:
Removed 😄